### PR TITLE
Scope/examples

### LINKED ISSUE #521: Write examples for NI-SCOPE

### PR DESCRIPTION
**Pull request by texasaggie97**
[X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).
~~[ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~
~~[ ] I've added tests applicable for this pull request~~
### What does this Pull Request accomplish?
* Add examples for niscope
* Remove placeholder example
### List issues fixed by this Pull Request below, if any.
* Fix #521 

### What testing has been done?
* Unit
* System
* Flake8 on new examples

### CODE REVIEW

File: src/niscope/examples/niscope_fetch.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+
+import argparse
+import niscope
+import pprint
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+parser = argparse.ArgumentParser(description='Outputs the specified voltage, then takes the specified number of voltage and current readings.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```
Comment (marcoskirsch) [inline src/niscope/examples/niscope_fetch.py]: Copied from NI-DCPower?

File: src/niscope/examples/niscope_fetch.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+parser.add_argument('-n', '--name', default='PXI1Slot2', help='Resource name of a National Instruments SMU')
```
Comment (marcoskirsch) [inline src/niscope/examples/niscope_fetch.py]: Digitizer

File: src/niscope/examples/niscope_fetch.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+parser.add_argument('-c', '--channels', default='0', help='Channel(s) to use')
+parser.add_argument('-l', '--length', default='1000', type=int, help='Measure record length')
+parser.add_argument('-v', '--voltage', default=1.0, type=float, help='Voltage range')
+args = parser.parse_args()
+
+with niscope.Session(args.name, False, False, 'Simulate=1, DriverSetup=Model:5164; BoardType:PXIe') as session:
```
Comment (marcoskirsch) [inline src/niscope/examples/niscope_fetch.py]: should use passed in device

File: src/niscope/examples/niscope_fetch.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    session.configure_vertical(args.voltage, 0.0, 0, 1.0, True)
+    session.configure_horizontal_timing(50000000, args.length, 50.0, 1, True)
+    with session.initiate():
+        wfm, wfm_infos = session[args.channels].fetch(1, args.length)
+    print('Number of samples acquired: {:,}\n'.format(len(wfm)))
+    row_format_g = '    {:<20}: {:,.6g}'
+    row_format_d = '    {:<20}: {:,}'
+    for i in range(len(wfm_infos)):
+        wfm_info = wfm_infos[i]
```
Comment (marcoskirsch) [inline src/niscope/examples/niscope_fetch.py]: It'd be nice if you can leverage __repr__ or __str__ of wfm_info here, somehow.

File: src/niscope/examples/niscope_fetch.py (hunk 1 above)
Comment (texasaggie97) [reply src/niscope/examples/niscope_fetch.py]: Changed

File: src/niscope/examples/niscope_fetch.py (hunk 2 above)
Comment (texasaggie97) [reply src/niscope/examples/niscope_fetch.py]: Changed

File: src/niscope/examples/niscope_fetch.py (hunk 3 above)
Comment (texasaggie97) [reply src/niscope/examples/niscope_fetch.py]: Removed option string

File: src/niscope/examples/niscope_fetch.py (hunk 4 above)
Comment (texasaggie97) [reply src/niscope/examples/niscope_fetch.py]: Added __str__() and used that.

**Pull request merged**